redwood-flexbox-engine/src/commonMain/kotlin/app/cash/redwood/properties.kt

```diff
@@ -0,0 +1,294 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@file:Suppress("MemberVisibilityCanBePrivate")
+
+package app.cash.redwood
+
+import kotlin.jvm.JvmInline
+import kotlin.math.abs
+
+/**
+ * This attribute controls the alignment of the flex lines in the flex container.
+ */
+@JvmInline
+public value class AlignContent private constructor(public val ordinal: Int) {
+
+  override fun toString(): String = when (ordinal) {
+    0 -> "FlexStart"
+    1 -> "FlexEnd"
+    2 -> "Center"
+    3 -> "SpaceBetween"
+    4 -> "SpaceAround"
+    5 -> "Stretch"
+    else -> "Unknown: $ordinal"
```

Comment:
```suggestion
    else -> throw AssertionError()
```
(For all these value classes below, too)